drivers/ata8520e/include/ata8520e_internals.h

```diff
@@ -60,6 +60,26 @@ extern "C" {
 #define ATA8520E_ATMEL_PA_MASK               (0x01)
 /** @} */
 
+/**
+ * @name Sigfox errors codes
+ * @{
+ */
+#define ATA8520E_SIGFOX_NO_ERROR             (0x00)
+#define ATA8520E_SIGFOX_TX_LEN_TOO_LONG      (0x30)
+#define ATA8520E_SIGFOX_RX_TIMEOUT           (0x3E)
+#define ATA8520E_SIGFOX_RX_BIT_TIMEOUT       (0x4E)
+/** @} */
+
+/**
+ * @name Sigfox2 errors codes
```

Comment:
What das Sigfox**2** mean? Is it worth a note here?

Reply:
IIRC, this is the name used in the datasheet